.circleci/scripts/deploy.sh

```diff
@@ -31,11 +36,14 @@ git add -A
 terminus -n auth:login --machine-token="$TERMINUS_MACHINE_TOKEN"
 
 if [ $CIRCLE_BRANCH == $SOURCE_BRANCH ]; then
-  git commit -m "automated static site deploy: ${CIRCLE_SHA1}" --allow-empty
+  git commit -m "automated static site deploy: ${GIT_COMMIT_MSG}" --allow-empty
   git push origin -f $SOURCE_BRANCH:$TARGET_BRANCH # push to gh-pages
   git push -f pantheon $SOURCE_BRANCH # push to pantheon master
+  git checkout $SOURCE_BRANCH
```

Comment:
is this redundant?  didn't we just push?

Reply:
Yeah you’re right. Redundant. 😞 Thanks for catching.